api/__tests__/app.test.js

```diff
@@ -44,11 +48,44 @@ describe.only('GET api/topics', () => {
 
 
 
+describe('GET api/articles', () => {
+    test('should return a 200 status', () => {
+        return supertest(app).get('/api/articles').expect(200)
+    
+    })
+    test('should return an array of topic objects with correct properties', () => {
+        return supertest(app).get('/api/articles').expect(200)
+        .then(({body}) => {
+            expect(body.length).toEqual(5); 
+            body.forEach((article) => {
+                    expect(article).toHaveProperty('author');
+                    expect(article).toHaveProperty('title');
+                    expect(article).toHaveProperty('article_id');
+                    expect(article).toHaveProperty('topic');
+                    expect(article).toHaveProperty('created_at');
+                    expect(article).toHaveProperty('votes');
+                    expect(article).toHaveProperty('article_img_url');
+                    expect(article).toHaveProperty('comments_count');
+                   
```

Comment:
it would be nice to have a test to see if an article coming back has the comment_count we expect to make sure our COUNT() is working as we expect